pkg/settings/settings_helper.go

```diff
@@ -67,7 +67,7 @@ func (target *BackupTarget) IsDefaultBackupTarget() bool {
 func InitVMForceResetPolicy() string {
 	policy := &VMForceResetPolicy{
 		Enable: true,
-		Period: 5 * 60, // 5 minutes
+		Period: 30, // 30 seconds
```

Comment:
the default originally was 300 seconds. just wanted to make sure if this is intentional?